[2020-06-17 10:46:29,419] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: email_example.hello_task 2020-06-17T10:41:40.978148+00:00 [queued]>
[2020-06-17 10:46:29,429] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: email_example.hello_task 2020-06-17T10:41:40.978148+00:00 [queued]>
[2020-06-17 10:46:29,429] {taskinstance.py:879} INFO - 
--------------------------------------------------------------------------------
[2020-06-17 10:46:29,429] {taskinstance.py:880} INFO - Starting attempt 1 of 1
[2020-06-17 10:46:29,429] {taskinstance.py:881} INFO - 
--------------------------------------------------------------------------------
[2020-06-17 10:46:29,438] {taskinstance.py:900} INFO - Executing <Task(PythonOperator): hello_task> on 2020-06-17T10:41:40.978148+00:00
[2020-06-17 10:46:29,440] {standard_task_runner.py:53} INFO - Started process 11728 to run task
[2020-06-17 10:46:29,494] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: email_example.hello_task 2020-06-17T10:41:40.978148+00:00 [running]> ip-172-31-76-14.ec2.internal
[2020-06-17 10:46:29,505] {python_operator.py:114} INFO - Done. Returned value was: Hello world!
[2020-06-17 10:46:29,516] {taskinstance.py:1065} INFO - Marking task as SUCCESS.dag_id=email_example, task_id=hello_task, execution_date=20200617T104140, start_date=20200617T104629, end_date=20200617T104629
[2020-06-17 10:46:31,985] {logging_mixin.py:112} INFO - [2020-06-17 10:46:31,985] {local_task_job.py:78} ERROR - Received SIGTERM. Terminating subprocesses
